docs/admin/postgres.mdx

```diff
@@ -23,19 +22,23 @@ intarray
 pg_stat_statements
 pg_trgm
 pgcrypto
+vector
+pg_vector
```

Comment:
When I list in my devenv what extensions I have installed it only mentions `vector`. Is `pg_vector` the old name and you include it here just in case? Should we call out this distinction just below?

Reply:
It should just be `vector` as that is what pg_vector shows up as.